packages/flutter/lib/src/cupertino/page.dart

```diff
@@ -0,0 +1,137 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+const double _kMinFlingVelocity = 1.0;  // screen width per second.
+const Color _kBackgroundColor = const Color(0xFFEFEFF4); // iOS 10 background color.
+
+// Used for iOS.
```

Comment:
It would be nice to have some dartdocs for this class now that it's public, but we can do that in another round if you prefer.